packages/lwc/README.md

```diff
@@ -0,0 +1,73 @@
+Lightning Web Components (LWC) is a blazing fast, enterprise-grade web components foundation for building user interfaces. LWC provides a simple authoring format for UI Components. LWC compiles this authoring format into low-level Web Component APIs. The `lwc` package is the main entry point for dependencies.
+
+-   Develop components quickly and declaratively using HTML, JavaScript, and CSS.
```

Comment:
one feedback that I have received multiple times is that the statically analyzable aspect of the framework, as a foundational part, is not mentioned anywhere. It only make reference to the declarative aspect of it... but I think we should mentioned somewhere that part of the deal is to author components in a way that the code can be statically analyzed .

Reply:
I think we can develop this out a bit for an update of the readme later on.